docs/functions/util.rst

```diff
@@ -143,40 +180,27 @@ These functions are designed to be used by themes.
 
    This function will do nothing if :attr:`LP_ENABLE_TITLE` is disabled.
 
-.. function:: _lp_formatted_title(title)
-
-   Sets the theme generated title to *title*. The input is escaped using
-   :func:`__lp_strip_escapes` to strip terminal formatting from being added to
-   the title.
-
-   This function will do nothing and return ``2`` if :attr:`LP_ENABLE_TITLE`
-   is disabled.
-
-   .. versionadded:: 2.0
-
-.. function:: _lp_raw_title(title)
+.. function:: _lp_version_greatereq([major, [minor, [patch, [string, [number]]]]])
 
-   Sets the theme generated title to *title*. The input is not escaped in any
-   way: if the input contains terminal formatting, use
-   :func:`_lp_formatted_title` instead.
+   Returns true (0) if Liquid Prompt version is greater than or equal to the the given version.
+   Returns 1 (false) if there is a *minor* or less version difference,
+   and 2 (false) if it is a *major* difference.
 
-   This function will do nothing and return ``2`` if :attr:`LP_ENABLE_TITLE`
-   is disabled.
+   See also :func:`_lp_version_string`.
 
-   .. versionadded:: 2.0
+   .. warning:: This only supports the following strings: "alpha", "beta" and "rc".
 
-.. function:: _lp_hash_color(str) -> var:lp_hash_color
+   .. versionadded:: 2.2
 
-   Colorize the given string with a color depending on its hash. The color is
-   chosen among: (green, yellow, blue, purple, cyan). Note that the red color is
-   not a candidate, as it should be reserved for alerts.
+.. function:: _lp_version_string([major, [minor, [patch, [string, [number]]]]]) -> var:lp_version
 
-   .. versionadded:: 2.2
+   Formats the given version number in a version string of the form:
+   "${major}.${minor}.${patch}-${string}-${number}"
```

Comment:
Semantic versioning allows for either, but I have been formatting the version with a period, as `${string}.${number}`, not with a hyphen. So `alpha.1`. Might as well be consistent.

Reply:
Should we keep the hyphen between patch and string?

Reply:
Yes, that is required by Semantic versioning, and what I have been doing. See tag [v2.1.0-rc.1](https://github.com/nojhan/liquidprompt/tree/v2.1.0-rc.1) for example.